lib/Model/Listing/AbstractListing.php

```diff
@@ -256,7 +256,7 @@ public function addConditionParam($key, $value = null, $concatenator = 'AND')
 
         $key = '('.$key.')';
         $ignore = true;
-        if (strpos($key, '?') !== false || strpos($key, ':') !== false) {
+        if ((strpos($key, '?') !== false || strpos($key, ':') !== false) && func_num_args() > 1) {
```

Comment:
```suggestion
        if (func_num_args() > 1 && (str_contains($key, '?') || str_contains($key, ':'))) {
```

Reply:
Thought again about `func_num_args()` this is not a good idea because then you could not do an OR condition likt this:
```php
$listing->addConditionParam('name is null', null, 'OR');
```

Now I look for a way to check if the `?` or `:` is within quotes - if it is, then it is not a prepared statement parameter.